### PR TITLE
Remove verify check for 0994

### PR DESCRIPTION
**Pull request by devin-mccurdy**
## Description
Update to 0994 config to make sure that users are only required to be logged, but not verified in order to access 0994

## Testing done
Tested locally

## Screenshots
N/A

## Acceptance criteria
- [x] Non-verified users should be able to access 0994

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/edu-benefits/tests/0994/e2e/vet-tec-helpers.js [hunk 1]
```diff
@@ -17,6 +17,29 @@ export const completeFormPage = (url, client, data, func) => {
...
 };
 
+export const completeApplicantInformation = (client, data) => {
+  const {
+    applicantFullName,
+    applicantSocialSecurityNumber,
+    dateOfBirth,
+  } = data;
+
+  client.fill(
```
Comment (zurbergram) [inline src/applications/edu-benefits/tests/0994/e2e/vet-tec-helpers.js]: you can chain these calls instead since each function returns the client object

**Review by zurbergram — APPROVED** [review]
LGTM

**Review by sonntag-adam — APPROVED** [review]

**Review by delli-gatti-michael — APPROVED** [review]
LTGM

**Comment by dmccurdybah** [comment]
@department-of-veterans-affairs/frontend-review-group Could I get a code review on this at your earliest convenience. Thank you!

**Review by jbalboni — APPROVED** [review]

**Pull request merged**